### ISSUE TITLE
Issue with plugin search bar cause E_ERROR with line 260

### ISSUE THREAD

**Issue by Nico13100** [woothemes/woocommerce#52931]
### Prerequisites

- [ ] I have carried out troubleshooting steps and I believe I have found a bug.
- [ ] I have searched for similar bugs in both open and closed issues and cannot find a duplicate.

### Describe the bug

Hi,
When I'm entering the name of a plugin in the search bar of the plugin page, nothing happens, it keeps to search on and on and in the same time, I receive an email saying : 

Détails de l’erreur
======================
Une erreur de type E_ERROR a été causée dans la ligne 260 du fichier /homepages/17/d840579272/htdocs/Martin-dumagny.myfactory.asia/wp-content/plugins/woocommerce/src/Packages.php. Message d’erreur : Uncaught Error: Call to a member function get_column_count() on null in /homepages/17/d840579272/htdocs/Martin-dumagny.myfactory.asia/wp-content/plugins/woocommerce/src/Packages.php:260
Stack trace:
#0 /homepages/17/d840579272/htdocs/Martin-dumagny.myfactory.asia/wp-includes/class-wp-hook.php(326): Automattic\WooCommerce\Packages::display_notice_for_merged_plugins()
#1 /homepages/17/d840579272/htdocs/Martin-dumagny.myfactory.asia/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters()
#2 /homepages/17/d840579272/htdocs/Martin-dumagny.myfactory.asia/wp-includes/plugin.php(517): WP_Hook->do_action()
#3 /homepages/17/d840579272/htdocs/Martin-dumagny.myfactory.asia/wp-admin/includes/class-wp-plugins-list-table.php(1491): do_action()
#4 /homepages/17/d840579272/htdocs/Martin-dumagny.myfactory.asia/wp-admin/includes/class-wp-plugins-list-table.php(711): WP_Plugins_List_Table->single_row()
#5 /homepages/17/d840579272/htdocs/Martin-dumagny.myfactory.asia/wp-admin/includes/class-wp-list-table.php(1712): WP_Plugins_List_Table->display_rows()
#6 /homepages/17/d840579272/htdocs/Martin-dumagny.myfactory.asia/wp-admin/includes/class-wp-list-table.php(1639): WP_List_Table->display_rows_or_placeholder()
#7 /homepages/17/d840579272/htdocs/Martin-dumagny.myfactory.asia/wp-admin/includes/ajax-actions.php(4834): WP_List_Table->display()
#8 /homepages/17/d840579272/htdocs/Martin-dumagny.myfactory.asia/wp-includes/class-wp-hook.php(324): wp_ajax_search_plugins()
#9 /homepages/17/d840579272/htdocs/Martin-dumagny.myfactory.asia/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters()
#10 /homepages/17/d840579272/htdocs/Martin-dumagny.myfactory.asia/wp-includes/plugin.php(517): WP_Hook->do_action()
#11 /homepages/17/d840579272/htdocs/Martin-dumagny.myfactory.asia/wp-admin/admin-ajax.php(192): do_action()
#12 {main}
 thrown

Do you have an idea of how I can fix that ?

Thank you for your help,

Nicolas

### Expected behavior

No response

### Actual behavior

No response

### Steps to reproduce

1. Go tu Plugin in backend
2. Enter a plugin name in search bar
3. It keeps on searching on and on
4. Then I receive the email error

### WordPress Environment

<details><summary>System Status Report</summary>

```
### WordPress Environment ###

WordPress address (URL): [Redacted]
Site address (URL): [Redacted]
WC Version: 9.4.1
Legacy REST API Package Version: L’extension Legacy REST API n’est pas installée sur ce site.
Action Scheduler Version: ✔ 3.8.2
Log Directory Writable: ✔
WP Version: 6.7
WP Multisite: –
WP Memory Limit: 1 Go
WP Debug Mode: –
WP Cron: ✔
Language: fr_FR
External object cache: –

### Server Environment ###

Server Info: Apache
PHP Version: 8.3.13
PHP Post Max Size: 64 Mo
PHP Time Limit: 50000
PHP Max Input Vars: 5000
cURL Version: 7.74.0
OpenSSL/1.1.1w

SUHOSIN Installed: –
MySQL Version: 10.6.15-MariaDB-log
Max Upload Size: 64 Mo
Default Timezone is UTC: ✔
fsockopen/cURL: ✔
SoapClient: ✔
DOMDocument: ✔
GZip: ✔
Multibyte String: ✔
Remote Post: ✔
Remote Get: ✔

### Database ###

[REDACTED]

### Post Type Counts ###

acoplw_badges: 2
attachment: 6895
bp-email: 21
custom_css: 3
custom-css-js: 1
cwginstocknotifier: 185
elementor_library: 12
forum: 3
is_search_form: 2
itsec-dash-card: 6
itsec-dashboard: 1
jet-popup: 1
mailpoet_page: 1
nav_menu_item: 218
oceanwp_library: 1
page: 14
post: 3
product: 3180
reply: 11
shop_order: 423
sp_wps_shortcodes: 1
topic: 2
wapf_product: 130
wcccf: 3
wccpf: 1
wccs_showcase: 1
wcpcsu-custom-post: 1
whatsapp-accounts: 1
wp_global_styles: 4
wp_navigation: 1
wpc_product_badge: 4
wpcode: 3

### Security ###

Secure connection (HTTPS): ✔
Hide errors from visitors: ✔

### Active Plugins (66) ###

Advanced Database Cleaner PRO: par Younes JFR. – 3.2.10
Advanced Product Fields Extended for WooCommerce: par StudioWombat – 2.7.25
Advanced Woo Search: par ILLID – 3.20
Akismet Anti-spam: Spam Protection: par Automattic - Équipe Anti-spam – 5.3.5
Back In Stock Notifier for WooCommerce | WooCommerce Waitlist Pro: par codewoogeek – 5.7.1
Barcode Scanner with Inventory & Order Manager - (business): par UkrSolution – 1.6.4
Classic Widgets: par Contributeurs et contributrices WordPress – 0.3
Code Snippets: par Code Snippets Pro – 3.6.5.1
Simple Custom CSS and JS: par SilkyPress.com – 3.49
Disable XML-RPC-API: par Neatma – 2.1.6
EAN for WooCommerce Pro: par WPFactory – 4.9.1
EAN Barcode Generator for WooCommerce: UPC, ISBN & GTIN Inventory: par WPFactory – 5.3.1
Elementor Pro: par Elementor.com – 3.25.3
Elementor: par Elementor.com – 3.25.7
GTranslate: par Solutions d’IA de traduction multilingue – 3.0.7
Health Check & Troubleshooting: par La communauté WordPress – 1.7.1
Imagify: par Imagify Image Optimizer – Optimize Images & Convert WebP & Avif – 2.2.3.2
Kadence WooCommerce Email Designer: par Kadence WP – 1.5.13
Lock User Account: par teknigar – 1.0.5
LoginPress: par LoginPress – 3.3.0
MailPoet Premium: par MailPoet – 5.3.0
MailPoet: par MailPoet – 5.3.7
Max Mega Menu - Pro Addon: par megamenu.com – 2.4
Max Mega Menu: par megamenu.com – 3.4
Ocean Elementor Widgets: par OceanWP – 2.4.5
Ocean Extra: par OceanWP – 2.4.2
Ocean Popup Login: par OceanWP – 2.2.0
Ocean Sticky Header: par OceanWP – 2.2.1
LoginWP (Formerly Peter's Login Redirect): par LoginWP Team – 3.0.8.6
Barcode Label Printing for WooCommerce and others plugins - (premium): par  – 3.4.11
ShiftNav Pro - Responsive Mobile Menu: par Chris Mavricos
SevenSpark – 1.7.2

Shortcode in Menus: par Gagan Deep Singh – 3.5.1
Simple Local Avatars: par 10up – 2.8.3
WooCommerce Split Orders: par Vibe Agency – 1.6.12
String Locator: par InstaWP – 2.6.6
Temporary Login: par Elementor.com – 1.2.0
UpdraftPlus - Backup/Restore: par TeamUpdraft
DavidAnderson – 2.24.11.26

User Switching: par John Blackbourn – 1.8.0
APF add-on: User by email condition: par Maarten Belmans – 1.0
Advanced Product Fields add-on: layered images: par StudioWombat – 2.1.6
YellowPencil Pro: par WaspThemes – 7.6.5
WooCommerce Weight Based Shipping: par weightbasedshipping.com – 6.1.0
Wish List for WooCommerce Pro: par WPFactory – 2.1.4
WooCommerce Address Book: par CrossPeak – 2.6.5
Cart Weight: par Octolize – 1.9.1
CartBounty - Save and recover abandoned carts for WooCommerce: par Streamline.lv – 8.3
Ultimate Order Combination: par Fahad Mahmood – 1.8.8
WooCommerce.com Update Manager: par Automattic – 1.0.3
WP Menu Cart: par WP Overnight – 2.14.10
PDF Invoices & Packing Slips for WooCommerce: par WP Overnight – 3.9.0
PDF Invoices & Packing Slips for WooCommerce - Professional: par WP Overnight – 2.15.11
PDF Invoices & Packing Slips for WooCommerce - Premium Templates: par WP Overnight – 2.21.10
HUSKY - Products Filter Professional for WooCommerce: par realmag777 – 3.3.6.3
Woocommerce Side Cart Premium: par XootiX – 3.1
WooCommerce: par Automattic – 9.4.1
StoreCustomizer: par Kaira – 2.5.7
Wordfence Security: par Wordfence – 8.0.1
Force Login: par Kevin Vess – 5.6.3
WP Mail SMTP: par WP Mail SMTP – 4.2.0
WP Rocket: par WP Media – 3.17.2.1
WPC Badge Management for WooCommerce (Premium): par WPClever – 3.0.4
WPC Countdown Timer for WooCommerce (Premium): par WPClever – 3.1.0
WPC Grouped Product for WooCommerce (Premium): par WPClever – 5.1.2
WPC Variations Radio Buttons for WooCommerce: par WPClever – 3.5.8
WPForms Lite: par WPForms – 1.9.2.2
WPS Hide Login: par WPServeur
NicolasKulka
wpformation – 1.9.17.1


### Inactive Plugins (0) ###


### Dropin Plugins () ###

advanced-cache.php: advanced-cache.php

### Settings ###

Legacy API Enabled: –
Force SSL: –
Currency: EUR (€)
Currency Position: right
Thousand Separator: .
Decimal Separator: ,
Number of Decimals: 2
Taxonomies: Product Types: external (external)
grouped (grouped)
simple (simple)
variable (variable)
woosg (woosg)

Taxonomies: Product Visibility: exclude-from-catalog (exclude-from-catalog)
exclude-from-search (exclude-from-search)
featured (featured)
outofstock (outofstock)
rated-1 (rated-1)
rated-2 (rated-2)
rated-3 (rated-3)
rated-4 (rated-4)
rated-5 (rated-5)

Connected to WooCommerce.com: ✔
Enforce Approved Product Download Directories: –
HPOS feature enabled: –
Order datastore: WC_Order_Data_Store_CPT
HPOS data sync enabled: –

### Logging ###

Enabled: ✔
Handler: Automattic\WooCommerce\Internal\Admin\Logging\LogHandlerFileV2
Retention period: 30 jours
Level threshold: –
Log directory size: 255 Ko

### WC Pages ###

Base de la boutique: #8 - /boutique/
Panier: #9 - /panier/ -  Contient le code court [woocommerce_cart]
Commander: #10 - /commander/ -  Contient le code court [woocommerce_checkout]
Mon compte: #11 - /mon-compte/
Conditions générales de vente et d’utilisation: ❌ Page non définie

### Theme ###

Name: OceanWP Child
Version: 3.2.1.1649135347
Author URL: https://oceanwp.org/about-me/
Child Theme: ✔
Parent Theme Name: OceanWP
Parent Theme Version: 4.0.2
Parent Theme Author URL: https://oceanwp.org/about-oceanwp/
WooCommerce Support: ✔

### Templates ###

Overrides: oceanwp/woocommerce/cart/mini-cart.php
oceanwp/woocommerce/content-single-product.php
oceanwp/woocommerce/loop/loop-start.php
oceanwp/woocommerce/single-product/product-image.php
oceanwp/woocommerce/single-product/title.php


### MailPoet ###

Sending Method: SMTP
Send all site's emails with: Méthode d’envoi par défaut de WordPress
Task Scheduler method: Action Scheduler
Cron ping URL: https://shop.loveasiangroup.com?mailpoet_router&endpoint=cron_daemon&action=ping

### Admin ###

Enabled Features: activity-panels
analytics
product-block-editor
coupons
core-profiler
customize-store
customer-effort-score-tracks
import-products-task
experimental-fashion-sample-products
shipping-smart-defaults
shipping-setting-tour
homescreen
marketing
mobile-app-banner
navigation
onboarding
onboarding-tasks
pattern-toolkit-full-composability
product-custom-fields
remote-inbox-notifications
remote-free-extensions
payment-gateway-suggestions
printful
shipping-label-banner
subscriptions
store-alerts
transient-notices
woo-mobile-welcome
wc-pay-promotion
wc-pay-welcome-page
launch-your-store

Disabled Features: product-data-views
experimental-blocks
minified-js
product-pre-publish-modal
settings
async-product-editor-category-field
product-editor-template-system
blueprint
reactify-classic-payments-settings
use-wp-horizon

Daily Cron: ✔ Next scheduled: 2024-11-20 06:11:29 +07:00
Options: ✔
Notes: 197
Onboarding: completed

### Action Scheduler ###

Annulé: 1
Oldest: 2024-11-06 09:00:25 +0700
Newest: 2024-11-06 09:00:25 +0700

Terminé: 7 620
Oldest: 2024-10-19 15:17:32 +0700
Newest: 2024-11-19 14:51:00 +0700

Échoué: 4 907
Oldest: 2022-11-10 21:51:04 +0700
Newest: 2024-11-18 10:38:33 +0700

En attente: 34
Oldest: 2024-11-19 14:47:19 +0700
Newest: 2024-12-07 11:42:50 +0700


### Status report information ###

Generated at: 2024-11-19 14:51:31 +07:00
```
</details>

### Isolating the problem

- [x] I have deactivated other plugins and confirmed this bug occurs when only WooCommerce plugin is active.
- [x] This bug happens with a default WordPress theme active, or [Storefront](https://woocommerce.com/products/storefront/).
- [x] I can reproduce this bug consistently using the steps above.

**Comment by rodelgc** [comment]
Hi @Nico13100 

While our goal is to address all the issues reported in this repository, GitHub should be treated as a place to report confirmed bugs only.

The type of issue you submitted looks like a support request. In order to confirm the bug, please reach out to our support channels below:

- Review [WooCommerce Self-Service Guide](https://docs.woocommerce.com/document/woocommerce-self-service-guide/) to see if the solutions listed there apply to your case;
- If you are a paying customer of WooCommerce, contact WooCommerce support by [opening a ticket or starting a live chat](https://woocommerce.com/contact-us/);
- Make a post on [WooCommerce community forum](https://wordpress.org/support/plugin/woocommerce/)

Also, plugin search is already outside of WooCommerce's functionalities. There might be something else in your site that causes this behavior in your site.

I am closing this issue for now as it seems to be a support request and not a bug. If I missed something, please leave a comment and we will take a second look.

**Comment by Shahzeen-amjad** [comment]
Hi! @rodelgc I'm experiencing the same issue on a site where only the WooCommerce plugin is enabled. Rolling back to version 9.3.3 fixed the issue, but updating the plugin back to version 9.4.2 triggers the same fatal error.

**Comment by ozgurkazancci** [comment]
Hello, I have the same issue.

"E_ERROR was caused in line 260 of the file plugins/woocommerce/src/Packages.php. Error message: Uncaught Error: Call to a member function get_column_count() on null in plugins/woocommerce/src/Packages.php:260"

Any idea?

**Comment by vaka2vaka** [comment]
Hello Regarding this problem here are the topics
https://wordpress.org/support/topic/woocommerce-blocks-search-by-installed-plugins/#post-18154223
The behavior is extremely strange. Switching delivery parameters, periodically, once out of ten, the search works. Then it stops working again. We changed everything – Hide delivery cost before entering the address – Delivery zones – and much more. If we understand the rules by which this works, we will let you know.
For now, the options for fixing it are:
1 Disable the plugin
2 Roll back to version 9 9 3
3 Select the line define(‘DISABLE_WP_CRON’, true);

**Comment by rootlabaunz** [comment]
Having same issue, the search on plugin page keeps going for ever, but i think the error is received when i deactivate any plugin.

An error of type E_ERROR was caused in line 260 of the file /home/aurootlab/public_html/wp-content/plugins/woocommerce/src/Packages.php. Error message: Uncaught Error: Call to a member function get_column_count() on null in /home/aurootlab/public_html/wp-content/plugins/woocommerce/src/Packages.php:260
Stack trace:

**Comment by PanosSynetos** [comment]
Hey everyone - I wanted to acknowledge that we are tracking this on 
- https://github.com/woocommerce/woocommerce/issues/53057

Our team is working on fixing this. Please follow the GitHub issue above for updates.

**Comment by rodelgc** [comment]
Thanks everyone for confirming the bug. Thank you @PanosSynetos for working on the fix. Reopening the issue now since the PR to fix this is still open.

**Comment by nathanss** [comment]
This is fixed by https://github.com/woocommerce/woocommerce/pull/53438 and will be released in 9.5.
